packages/wp-dashboard/src/effects/useMediaOptimization.js

```diff
@@ -51,7 +51,11 @@ export default function useMediaOptimization() {
     trackEvent('video_optimization_optin', {
       status: mediaOptimization ? 'off' : 'on',
     });
-    showSnackbar({ message: SUCCESS.SETTINGS.UPDATED, dismissible: true });
+    showSnackbar({
+      'aria-label': SUCCESS.SETTINGS.UPDATED,
```

Comment:
there's a prop type error that was showing up in some snackbars. Same as below.

Reply:
Awesome. I've been seeing this in the console. thanks for cleaning up